packages/customize-widgets/src/components/inspector/index.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState, createPortal } from '@wordpress/element';
+import { BlockInspector } from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+export { default as BlockInspectorButton } from './block-inspector-button';
+
+const container = document.createElement( 'div' );
+
+function InspectorSectionMeta( { collapse } ) {
+	return (
+		<div className="customize-section-description-container section-meta">
+			<div className="customize-section-title">
+				<button // eslint-disable-line react/forbid-elements
```

Comment:
Ahh I've added that in a prior commit but somehow accidentally removed that 😅.

> I assume it's because we don't want the class names that Button adds since this has to match Customizer styling?

Exactly.